README.md

```diff
@@ -1 +1,73 @@
-# equalizer
\ No newline at end of file
+# Wolox Equalizer
+[![circle-ci](https://img.shields.io/circleci/project/github/Wolox/wolox-equalizer.svg)](https://circleci.com/gh/Wolox/wolox-equalizer)
+[![npm](https://img.shields.io/npm/v/wolox-equalizer.svg)](https://www.npmjs.com/package/wolox-equalizer)
+
+[![FEArmy](./assets/FEA_icon.png)](https://github.com/orgs/Wolox/teams/front-end-army/members)
+## Features
+
+- Reset the default styles of the browser.
+- Contains our Wolox standards for normalizing front-end projects.
+
+## Installation
+
+#### npm
+```bash
+npm install --save wolox-equalizer
+```
+
+#### yarn
+```bash
+yarn add wolox-equalizer
+```
+
+## Usage
+
+1- Include the equalizer.scss to the root js file
+
+```js
+import 'wolox-equalizer/equalizer.scss';
+```
+
+
+## Inital color value
+
+1- All texts use **#717171** color by default.
+
+
+## Contributing
```

Comment:
Add a separate `CONTRIBUTING.md` file, it suits better with Gihub standards

Reply:
We took the [redux recompose readme](https://github.com/Wolox/redux-recompose) format